examples/verify_addresses_from_csv/verify.py

```diff
@@ -46,55 +45,37 @@
     sys.exit(1)
 
 # output csv names
-success_filename = os.path.join(output_dir, 'success.csv')
-errors_filename = os.path.join(output_dir, 'errors.csv')
+verifications_filename = os.path.join(output_dir, 'verifications.csv')
```

Comment:
Not quite. The API will always return a 200 response. There are 5 different `deliverability` values that can be returned but I don't think we should consider any of those as errors but rather some as invalid addresses. However, these are still verifications.